bindings/python/src/model/representation/core/brep.h

```diff
@@ -172,82 +161,76 @@ namespace geode
                     return components;
                 },
                 pybind11::return_value_policy::reference )
-            .def( "nb_embedded_corner_surfaces",
+            .def( "nb_embedding_corner_surfaces",
```

Comment:
again, thanks to the respective dimensions, we guess who is embedding who but it is not obvious to which object the adjective is related to. 
one suggestion : embedded_corner_inside_surfaces -> would give the corner
another suggestion: embedding_surfaces_of_corner -> would give the range of surfaces

Reply:
@panquez an opinion on this? I agree that it is not obvious at all.

Reply:
The best name would be nb_embedding_surfaces. But since we cannot overload these functions we should specified on which type of objects they are applied. IMO, Marion's suggestion `embedding_surfaces_of_corner` is the best we can do

Reply:
done